requirements.txt

```diff
@@ -1,5 +1,7 @@
 argparse>=1.2.1
+freezegun==0.3.9
 Jinja2>=2.8
+pytz==2017.2
```

Comment:
i _think_ these modules also need to be added to `setup.py`

Reply:
they do